### PR TITLE
About page

### PR DESCRIPTION
**Pull request by kailamjeter**
## Description

Addresses: Adds content for about us page

Adds 'About us' page view. Note that values for css in mocks seemed a bit off when applied so went on what looked best to me -- open to comments on how things look!

## Testing and Reviewing

ran locally + navigated to the page using footer

## screenshot
Screen Shot 2022-08-19 at 10 14 26 AM

### CODE REVIEW

File: client/src/views/AboutUsView.vue [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  height: 100%;
+  background-color: $light-blue-800;
+  background-image: url('~@/assets/media/about-us-image.png');
+  background-size: cover;
+  background-repeat: no-repeat;
+  color: $white;
+}
+
+.h1-header {
+  width: 169 * $spacing-xs;
```
Comment (beekley) [inline client/src/views/AboutUsView.vue:45]: nit: multiplying by a factor here seems to defeat the purpose of using a common value. Would [`max-width`](https://github.com/BlueConduit/copper/blob/main/packages/copper/scss/01_settings/_design-tokens.scss#L55) work instead?

File: client/src/views/AboutUsView.vue [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+}
+
+.h1-header-xxl {
+  font-weight: 600;
+  line-height: 15 * $spacing-xs;
```
Comment (beekley) [inline client/src/views/AboutUsView.vue:50]: nit: [`spacing-xl`](https://github.com/BlueConduit/copper/blob/main/packages/copper/scss/01_settings/_design-tokens.scss#L52) is about the same as this value.

**Review by beekley — APPROVED** [review]
LGTM. If Copper doesn't have good values to use, I'd rather just use a literal value (e.g. `3.75rem` than coerce something from Copper (e.g. `15 * spacing-xs`) so we don't have to look up a file and do math to figure out what it means.

We can always copy it over to Copper in the future for code reuse.

File: client/src/assets/messages/about_us.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
+export class AboutUs {
+  static HEADER = 'About us';
+  static BODY_1 =
+    'Leadout is a tool created by BlueConduit. BlueConduit is a water analytics company that has ' +
+    'developed cutting-edge, predictive machine learning software to locate lead service lines, ' +
+    'empowering local officials and their engineering partners with the information to ' +
+    'efficiently remove those pipes.';
+  static BODY_2 =
```
Comment (justinebreuch) [inline client/src/assets/messages/about_us.ts:8]: nit: make these more descriptive var names

File: client/src/assets/styles/global.scss [hunk 4]
```diff
@@ -1,9 +1,19 @@
...
+$light-blue-800: #0576BD;
+
 /** Typography */
 
 .h1-header {
   font-size: 24px;
 }
 
+.h1-header-xxl {
+  font-size: 80px;
```
Comment (justinebreuch) [inline client/src/assets/styles/global.scss:14]: This is reall big - even bigger than the logo I think. I think it's fine to keep for now I'm just adding a task to lead out to standardize these fonts. Can you do the same thing you did elsewhere 4px * xxx?

File: client/src/views/AboutUsView.vue (hunk 2 above)
Comment (justinebreuch) [reply client/src/views/AboutUsView.vue:50]: Is this the same xxl as is in the global styles?

File: client/src/views/AboutUsView.vue [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+  text-transform: capitalize;
+}
+
+.text {
+  display: flex;
```
Comment (justinebreuch) [inline client/src/views/AboutUsView.vue:55]: nit: container-column class or mixin

**Review by justinebreuch — APPROVED** [review]

File: client/src/views/AboutUsView.vue (hunk 2 above)
Comment (kailamjeter) [reply client/src/views/AboutUsView.vue:50]: Removed this but changed elsewhere

File: client/src/assets/styles/global.scss (hunk 4 above)
Comment (kailamjeter) [reply client/src/assets/styles/global.scss:14]: Made 56px instead (what I previously added as h1-header-xl). Will do a followup PR to remove from landing page styles I submitted but yes will fix here. Thanks!

File: client/src/views/AboutUsView.vue (hunk 5 above)
Comment (kailamjeter) [reply client/src/views/AboutUsView.vue:55]: Done

File: client/src/views/AboutUsView.vue (hunk 1 above)
Comment (kailamjeter) [reply client/src/views/AboutUsView.vue:45]: Changed to max-width and just a px value. Left a TODO to add a var for these values. Thanks!

File: client/src/assets/messages/about_us.ts (hunk 3 above)
Comment (kailamjeter) [reply client/src/assets/messages/about_us.ts:8]: Done

**Pull request merged**